server/data/serviceCatalogue/serviceCatalogue.yaml

```diff
@@ -0,0 +1,352 @@
+      order: 8
+      disabled: false
+    - name: hmpps-education-employment-api
+      label: HMPPS Education Employment
+      url: https://education-employment-api-dev.hmpps.service.justice.gov.uk
+      order: 9
+      disabled: false
+    - name: hmpps-hdc-api
+      label: HMPPS HDC
+      url: https://hdc-api-dev.hmpps.service.justice.gov.uk
+      order: 10
+      disabled: false
+    - name: hmpps-incidents-reporting-api
+      label: HMPPS Incident Reporting
+      url: https://incident-reporting-api-dev.hmpps.service.justice.gov.uk
+      order: 11
+      disabled: false
+    - name: hmpps-incentives-api
+      label: HMPPS Incentives
+      url: https://incentives-api-dev.hmpps.service.justice.gov.uk
+      order: 12
+      disabled: false
+    - name: hmpps-interventions-service
+      label: HMPPS Interventions Service
+      url: https://hmpps-interventions-service-dev.apps.live-1.cloud-platform.service.justice.gov.uk
+      order: 13
+      disabled: false
+    - name: hmpps-manage-adjudications-api
+      label: HMPPS Manage Adjudications
+      url: https://manage-adjudications-api-dev.hmpps.service.justice.gov.uk
+      order: 14
+      disabled: false
+    - name: hmpps-non-associations-api
+      label: HMPPS Non Associations
+      url: https://non-associations-api-dev.hmpps.service.justice.gov.uk
+      order: 15
+      disabled: false
+    - name: hmpps-offender-categorisation-api
+      label: HMPPS Offender Categorisation
+      url: https://hmpps-offender-categorisation-api-dev.hmpps.service.justice.gov.uk
+      order: 16
+      disabled: false
+    - name: hmpps-restricted-patients-api
+      label: HMPPS Restricted Patients
+      url: https://restricted-patients-api-dev.hmpps.service.justice.gov.uk
+      order: 17
+      disabled: false
+    - name: hmpps-resettlement-passport-api
+      label: HMPPS Resettlement Passport
+      url: https://resettlement-passport-api-dev.hmpps.service.justice.gov.uk
+      order: 18
+      disabled: false
+    - name: hmpps-uof-data-api
+      label: HMPPS Uof Data
+      url: https://hmpps-uof-data-api-dev.hmpps.service.justice.gov.uk
+      order: 19
+      disabled: false
+    - name: keyworker-api
+      label: Keyworker
+      url: https://keyworker-api-dev.prison.service.justice.gov.uk
+      order: 20
+      disabled: false
+    - name: make-recall-decision-api
+      label: Make Recall Decision
+      url: https://make-recall-decision-api-dev.hmpps.service.justice.gov.uk
+      order: 21
+      disabled: false
+    - name: offender-case-notes
+      label: Offender Case Notes
+      url: https://dev.offender-case-notes.service.justice.gov.uk
+      order: 22
+      disabled: false
+    - name: offender-management-allocation-manager
+      label: Offender Management Allocation Manager
+      url: https://test.moic.service.justice.gov.uk
+      order: 23
+      disabled: false
+  - environment: preprod
```

Comment:
would have thought the environment(s) could be part of the group eg
dev: enabled
preprod: disabled
prod: disable

Reply:
this file is the only source of service urls and there are different urls for each service